XenModel/Messages.resx

```diff
@@ -12784,7 +12784,7 @@ A {1} user cannot alter the Workload Balancing settings.</value>
     <value>One or more hosts in the pool do not have an appropriate license.</value>
   </data>
   <data name="WLB_ERROR_4021" xml:space="preserve">
-    <value>WLB could not start a [XenServer] session with supplied invalid credentials.</value>
+    <value>WLB could not connect to [XenServer] because the supplied credentials were invalid.</value>

```

Comment:
Hmm, ok, maybe, but then it should probably be "to the [XenServer] host", just "connect to [XenServer]" sounds a bit weird to me.